__tests__/hooks/useContentPoll.test.js

```diff
@@ -55,7 +55,7 @@ describe('Content poll function should run correctly', () => {
       result.current.setIsAlreadyPolling(false);
     });
     expect(result.current.showSetResultsButton(expiredDate)).toBeFalsy();
-    expect(result.current.showSetResultsButton(notExpiredDare)).toBeTruthy();
+    expect(result.current.showSetResultsButton(notExpiredDare)).toBeFalsy();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the expected result of `showSetResultsButton(notExpiredDare)` from `toBeTruthy()` to `toBeFalsy()` might indicate a change in the function's behavior. Ensure that this is the intended behavior and that it doesn't break any existing functionality.
```diff
-    expect(result.current.showSetResultsButton(notExpiredDare)).toBeTruthy();
+    expect(result.current.showSetResultsButton(notExpiredDare)).toBeFalsy();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->